src/ctia/stores/es/crud.clj

```diff
@@ -92,19 +87,19 @@
                     (build-create-result model coerce-fn)))
                 (remove-es-actions items) models)}))
 
-
-(s/defn get-docs-with-indices
+(defn get-docs-with-indices
   "Retrieves a documents from a search \"ids\" query. It enables to retrieves
  documents from an alias that points to multiple indices.
 It returns the documents with full hits meta data including the real index in which is stored the document."
-  [{:keys [conn index]} :- ESConnState
-   ids :- [s/Str]
+  [{:keys [conn index] :as _conn-state}
+   ids
```

Comment:
Why did you remove the schemas here? They looked fine.

Reply:
probably a trace of some dev fight with myself :-)